examples/docker-bookinfo.yaml

```diff
@@ -56,28 +58,19 @@ ratings-v1:
 reviews-v1:
   image: amalgam8/a8-examples-bookinfo-reviews-sidecar:v1
   environment:
-    - A8_REGISTRY_URL=http://registry:8080
     - A8_SERVICE=reviews:v1
```

Comment:
Why do we need this environment variable?
IF its for nginx, then, we are already constructing it in sidecar. I am not sure if the reviews Java code is reading the A8_SERVICE environment variable.